Epam.TestAutomation/Core/Browser/Browser.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections.ObjectModel;
+using System.Runtime.CompilerServices;
+using Epam.TestAutomation.Core.Helper;
+using Epam.TestAutomation.Core.ScreenShot;
+using Epam.TestAutomation.Core.Utils;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+namespace Epam.TestAutomation.Core.Browser;
+
+public class Browser
+{
+    private IWebDriver driver { get;}
+    //private Actions action { get; set; }
+    IJavaScriptExecutor jse => (IJavaScriptExecutor)driver;
+
+    public Browser(IWebDriver driver)
+    {
+        this.driver = driver;
+    }
+
+    public void ScrollToElements(IWebElement element)
+    {
+        Logger.Info("The page is scrolled to the specified element");
+        IJavaScriptExecutor jse = (IJavaScriptExecutor)driver;
+        jse.ExecuteScript("window.scrollTo(0, document.body.scrollHeight)", element);
+    }
+
+    public void ClickElement(By locator)
```

Comment:
Why did you add Click and SendKeys to Browser class?

Reply:
Because they for for By type elements